lib/agent/grpc/request.go

```diff
@@ -26,6 +26,19 @@ func storeAttackStats(req *protos.AttackDetected) {
 	}
 }
 
+func storeMonitoredListsMatches(m *map[string]int, lists []string) {
+	if *m == nil {
+		*m = make(map[string]int)
```

Comment:
maybe return? safer without modification?

Reply:
We need to initialise `*m` so that we can modify it, wasn't clear to me at first.